plugins/modules/aci_route_control_profile.py

```diff
@@ -0,0 +1,297 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "certified"}
+
+DOCUMENTATION = r"""
+---
+module: aci_route_control_profile
+short_description: Manage Route Control Profile objects (rtctrl:Profile)
+description:
+- Manage Route Control Profile on Cisco ACI fabrics.
+options:
+  tenant:
+    description:
+    - Name of an existing tenant.
+    type: str
+    aliases: [ tenant_name ]
+  l3out:
+    description:
+    - Name of an existing L3Out.
+    type: str
+    aliases: [ l3out_name ]
+  description:
+    description:
+    - Description for the Route Control Profile.
+    type: str
+    aliases: [ descr ]
+  profile:
+    description:
+    - Name of the Route Control Profile
+    type: str
+    aliases: [ profile_name, route_control_profile ]
+  prof_type:
+    description:
+    - Type of the profile
+    type: str
+    choices: [ combinable, global ]
+  autocontinue:
+    description:
+    - Autocontinue
```

Comment:
Please add 
```
- The APIC defaults to false on when unset during creation.

Reply:
Updated. Bear in mind that the autocontinue parameter isn't documented very well in the APIC docs, so we don't have much of a description to give it (I've only really included it because it is an option in the model).